fm-overlays.php

```diff
@@ -30,12 +30,24 @@
 define( 'FM_OVERLAYS_VERSION', '0.0.1' );
 
 /**
- * Filesystem path to Ad Layers.
+ * Assets Version number.
+ *
+ * @var string
+ */
+define( 'FM_OVERLAYS_ASSETS_VERSION', '0.0.1' );
+
+/**
+ * Filesystem path to Fm Overlays.
  *
  * @var string
  */
 define( 'FM_OVERLAYS_PATH', plugin_dir_path( __FILE__ ) );
 
+/**
+ * URL for assets.
+ */
+define( 'FM_OVERLAYS_ASSET_DIR', plugin_dir_url( __FILE__ ) );
```

Comment:
For readability, it seems a little confusing that this is described as a URL but the constant is `_DIR`.